Config/config.cs

```diff
@@ -0,0 +1,27 @@
+namespace qa_dotnet_cucumber.Config
+{
+    public class TestSettings
```

Comment:
so you need to provide a separate json file to each test! this ensures tests will not use each others test data